pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/TransformFunctionFactory.java

```diff
@@ -49,6 +50,10 @@ private TransformFunctionFactory() {
           put(SubtractionTransformFunction.FUNCTION_NAME.toLowerCase(), SubtractionTransformFunction.class);
           put(MultiplicationTransformFunction.FUNCTION_NAME.toLowerCase(), MultiplicationTransformFunction.class);
           put(DivisionTransformFunction.FUNCTION_NAME.toLowerCase(), DivisionTransformFunction.class);
+          put(SqlKind.PLUS.name().toLowerCase(), AdditionTransformFunction.class);
```

Comment:
It’s probably better to change the function name in calcite sql parser to match Pinot functions. This will restrict calcite classes to parsing

Reply:
I cannot refer `AdditionTransformFunction` in `pinot-core` from calcite parser.
Another option here is that I can create a `PlusTransformFunction` extends `AdditionTransformFunction`, by just overriding the function name to `plus`